debian/control

```diff
@@ -53,9 +53,6 @@ Section: introspection
 Depends:
  ${gir:Depends},
  ${misc:Depends},
-Provides: gir1.2-ostree
-Conflicts: gir1.2-ostree
-Breaks: gir1.2-ostree
```

Comment:
I suppose there's no harm in removing this and the libostree-1 back compat names. It's been long enough.